src/utils/crc32.ts

```diff
@@ -0,0 +1,19 @@
+const POLYNOMIAL = 0xedb88320; // Polynomial used in CRC32 calculation
```

Comment:
Suggestion: if you have more details on the selection here it might be helpful to record them as a comment in case we need to make adjustments in the future.

Reply:
I'll add a note which standard we are using